documentation/Kubecheck.md

```diff
@@ -699,6 +701,22 @@ The test checks the AppArmor configuration. It has several modes: `enforce`, `co
 This test checks that the configuration of Kubernetes audit policy is actual
 and matches the effectively resolved configuration from the inventory.
 
+##### 231 Audit Daemon Rules
+
+*Task*: `system.audit.rules`
+
+This test compares the Audit rules on the nodes
+with the rules specified in the inventory or with the default rules.
+If the configured rules are not presented, the test fails.
+
+##### 232 Kernel Parameters Configuration
+
+*Task*: `system.sysctl.config`
+
+This test compares the Kernel parameters on the nodes
```

Comment:
Hi @shmo1218,
I think Linux kernel is written from lover case letter.
E.g. https://man7.org/linux/man-pages/man8/sysctl.8.html

Reply:
Hi, @ilia1243,

I have rolled back the format for "kernel" to its original form.